CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ### Added
 
+- :fireworks: `server_tokens` and `latency_tokens` configuration fields.
+  Check the [0.10 Configuration Guide](https://getkong.org/docs/0.10.x/configuration/#server_tokens)
+  to learn more.
+  [#2259](https://github.com/Mashape/kong/pull/2259)
```

Comment:
We could explain a bit more what this change is about and respect the 80 cols limit as well. Maybe:
```
- Ability to hide Kong-specific response headers. Two new configuration fields:
  `server_tokens` and `latency_tokens` will respectively toggle whether the `Server`
  and `X-Kong-*-Latency` headers should be sent to downstream clients.
``` 

Also in general, there is no need to point to the configuration guide for new config fields, since the guide does not elaborate more on properties than the `kong.conf.default` file,which already has a comment for each property.